lib/cli-update-connection.ts

```diff
@@ -119,7 +119,7 @@ export default function CliUpdateConnection() {
             };
 
             client.gateways.patchDatasource(settings.collection, settings.workspace, datasource.gatewayId, datasource.id, delta, (err, patchResult) => {
-                if (err) {
```

Comment:
Agreed - I'll put together a new PR that handles this.  I just noticed some other potential unhanded error conditions as well.